experimental/rules_python/wheelmaker.py

```diff
@@ -254,7 +268,9 @@ def main():
                     python_tag=arguments.python_tag,
                     abi=arguments.abi,
                     platform=arguments.platform,
-                    outfile=arguments.out) as maker:
+                    outfile=arguments.out,
+                    package_root_path=arguments.package_root_path,
```

Comment:
If I recall correctly having trailing comma on argument list is not supported on all python versions. Please remove it.

Reply:
Done